tests/clients/test_dapr_grpc_client.py

```diff
@@ -198,29 +199,20 @@ def test_get_save_delete_state(self):
         resp = dapr.get_state(store_name="statestore", key=key)
         self.assertEqual(resp.data, b'')
 
-    def test_save_get_states(self):
+    def test_transaction_then_get_states(self):
```

Comment:
am I missing something or why is save_state test removed?

Reply:
It is in another test. I decided to bundle it in the bulk get test since it seems they fit well together.